pkg/model/dev.go

```diff
@@ -111,6 +112,14 @@ type Dev struct {
 	SSHServerPort        int                   `json:"sshServerPort,omitempty" yaml:"sshServerPort,omitempty"`
 }
 
+// BuildInfo represents the build info to generate an image
+type BuildInfo struct {
```

Comment:
Will this be overridable via the CLI?

Reply:
I can add it as part of this PR

Reply:
only if we have a customer ask, if not, let's keep it simple and wait?

Reply:
Yeap, that is why I didn't add them.
Also, those parameters shouldn't change in the okteto workflow